src/borg/testsuite/archiver.py

```diff
@@ -57,6 +57,9 @@ def exec_cmd(*args, archiver=None, fork=False, exe=None, **kw):
         except subprocess.CalledProcessError as e:
             output = e.output
             ret = e.returncode
+        except SystemExit as e:
+            output = ''
+            ret = e.code
```

Comment:
Hmm, since that only applies to in-source (vs. binary) testing this change should rather go into [exec_cmd](https://github.com/borgbackup/borg/blob/master/src/borg/testsuite/archiver.py#L61)